src/tribler/ui/src/pages/Popular/index.tsx

```diff
@@ -57,12 +57,32 @@ export default function Popular() {
     const [open, setOpen] = useState<boolean>(false)
     const [torrents, setTorrents] = useState<Torrent[]>([])
     const [torrentDoubleClicked, setTorrentDoubleClicked] = useState<Torrent | undefined>();
+    const [request, setRequest] = useState<string>("");
 
     useInterval(async () => {
         const popular = await triblerService.getPopularTorrents(true);
         setTorrents(filterDuplicates(popular, 'infohash'));
+        const remoteQuery = await triblerService.searchTorrentsRemote('', true);
+        setRequest(remoteQuery.request_uuid);
     }, 5000, true);
 
+    useEffect(() => {
```

Comment:
Why reset the event listener every time the request changes?

Reply:
This was copied over from the search endpoint. I don't know of a better way.

Reply:
Ah, I missed that. It should be OK to pass an empty list as a parameter (like we do on the downloads page). But since we're doing elsewhere too, I'm fine with it as-is. Maybe there's a reason for it.

Reply:
I can try it out and see what happens. I'll report back.

Reply:
For some reason, replacing `[request]` with `[]` causes `data.uuid !== request` to always evaluate to `false`. So, you never see any results (don't ask me why).